arwenmandos/elrondProtectedStorage.go

```diff
@@ -1,7 +1,7 @@
 package arwenmandos
 
-// ElrondProtectedKeyPrefix prefixes all Elrond reserved storage. Only the protocol can write to keys starting with this.
-const ElrondProtectedKeyPrefix = "ELROND"
+// ProtectedKeyPrefix prefixes all Elrond reserved storage. Only the protocol can write to keys starting with this.
```

Comment:
```suggestion
// ProtectedKeyPrefix prefixes all MultiversX reserved storage. Only the protocol can write to keys starting with this.
```

Reply:
The content of the files will be changed as little as possible at this stage. But subsequent PRs will bring comprehensive changes indeed, including comments.

#9 is an example PR with comprehensive changes, but it will also be split into multiple PRs.